xds/src/main/proto/centraldogma/xds/group/v1/xds_group.proto

```diff
@@ -0,0 +1,62 @@
+// Copyright 2024 LINE Corporation
+//
+// LINE Corporation licenses this file to you under the Apache License,
+// version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at:
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+syntax = "proto3";
+
+package centraldogma.xds.group.v1;
+
+option java_multiple_files = true;
+option java_outer_classname = "XdsGroupProto";
+option java_package = "com.linecorp.centraldogma.xds.group.v1";
+
+import "google/api/annotations.proto";
+import "google/api/client.proto";
+import "google/api/field_behavior.proto";
+import "google/protobuf/empty.proto";
+
+// An XdsGroupService provides methods to manage groups.
+service XdsGroupService {
+
+  // Creates a new group.
+  rpc CreateGroup(CreateGroupRequest) returns (Group) {
+    option (google.api.http) = {
+      post: "/api/v1/xds/groups"
+      body: "group"
+    };
+  }
+
+  // Deletes an group.
+  rpc DeleteGroup(DeleteGroupRequest) returns (google.protobuf.Empty) {
```

Comment:
Note: This is a red flag personally since I think there is always the possibility that additional fields will need to be added. Having said this, I guess it's fine to try adhering to the AIP like you proposed for now.

Reply:
> This is a red flag personally since I think there is always the possibility that additional fields will need to be added. 

Would you mind explaining which fields are likely to be added later?
If that's reasonable, I will define a message and return it instead of the `Empty`.

Reply:
Let's just follow the AIP for now since the point is to ease these types of discussions.